<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileModify</class>
 <widget class="QWidget" name="FileModify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileModify</string>
  </property>
  <layout class="QVBoxLayout" name="vLayout">
   <item>
    <widget class="QGroupBox" name="grpPath">
     <property name="title">
      <string>指定文件所在目录</string>
     </property>
     <layout class="QGridLayout" name="gLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblPath">
        <property name="text">
         <string>目录路径：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="linPath">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="btnPath">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>文件类型：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbType">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>*.*</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>*.c *.h *.s</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabOper">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabName">
      <attribute name="title">
       <string>更改文件名称</string>
      </attribute>
      <layout class="QHBoxLayout" name="hLayoutN">
       <item>
        <widget class="QLabel" name="lblSrcN">
         <property name="text">
          <string>将文件名称中的</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linSrcN">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDstN">
         <property name="text">
          <string>替换为</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linDstN">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="hSpacerN">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="chkCaseN">
         <property name="text">
          <string>区分大小写</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExecN">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>执行</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabText">
      <attribute name="title">
       <string>替换文件内容</string>
      </attribute>
      <layout class="QHBoxLayout" name="hLayoutT">
       <item>
        <widget class="QLabel" name="lblSrcT">
         <property name="text">
          <string>将文件内容中的</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linSrcT">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDstT">
         <property name="text">
          <string>替换为</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linDstT">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="hSpacerT">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="chkCaseT">
         <property name="text">
          <string>区分大小写</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExecT">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>执行</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCode">
      <attribute name="title">
       <string>变换文件编码</string>
      </attribute>
      <layout class="QHBoxLayout" name="hLayoutC">
       <item>
        <widget class="QLabel" name="lblSrcC">
         <property name="text">
          <string>将文件内容编码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbSrcC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>133</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>GBK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UTF-8</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDstC">
         <property name="text">
          <string>转换为</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbDstC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>133</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>UTF-8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GBK</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="hSpacerC">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnExecC">
         <property name="text">
          <string>执行</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLine">
      <attribute name="title">
       <string>变换文件换行符</string>
      </attribute>
      <layout class="QHBoxLayout" name="hLayoutL">
       <item>
        <widget class="QLabel" name="lblSrcL">
         <property name="text">
          <string>将文件内容换行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbSrcL">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>133</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>\r\n</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>\n</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDstL">
         <property name="text">
          <string>转换为</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbDstL">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>133</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>\n</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>\r\n</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="hSpacerL">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnExecL">
         <property name="text">
          <string>执行</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="txtInfo">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
